packages/blade/src/components/BladeProvider/__tests__/paymentLightTheme/paymentLightTheme.web.ts

```diff
@@ -2,6 +2,7 @@ import type { Theme } from '~components/BladeProvider';
 import { paymentTheme } from '~tokens/theme';
 
 const paymentLightTheme: Theme = {
+  name: 'paymentTheme',
```

Comment:
Should we also include the color scheme just like in file name?

Reply:
colorScheme is not a static value. It can be accessed inside react component only using `useTheme`.

Token is a static object which has things like `onLight`, `onDark`. Now while we are defining the token object, we can't tell what is going to be the colorScheme.

If someone wants to watch over colorScheme, they can do `const { colorScheme } = useTheme()` and watch over that variable since it is just string.

Reply:
So the file is named `paymentLightTheme` but we're naming the theme `paymentTheme`. I feel both should be same. Say for `paymentDarkTheme`, would we use the same theme name? In terms of look and feel the visuals would be different 🤔

Reply:
Oh this is limited for testing. Its basically a testing utility that returns paymentTheme with all the `onLight` tokens manually.
<img width="454" alt="image" src="https://user-images.githubusercontent.com/30949385/222669545-51de8c9e-ead9-49cc-81f0-c6a158acd90c.png">

So we can use this utility when we want to pass theme object to some function (e.g. in a lot of helper functions of Box I am passing theme object so in testing, I use this file)

But the actual theming tokens send both `onLight` and `onDark` tokens and then decide which one to use based on `colorScheme` later.

Reply:
We might need some restructuring there though. I plan to explore this in my performance debugging task. We currently have 2x the theme bundle than it ideally should be 🤔